.jshintrc

```diff
@@ -30,6 +30,7 @@
     "define": true,
     "require": true,
     "Chromath": false,
-    "setImmediate": true
+    "setImmediate": true,
+    "$": false
```

Comment:
well I think you misunderstood this :) the jshint error is CORRECTLY complaining that you are using an undefined variable. the correct thing would be to require jquery as a dependency in the top of the js file, not to remove the validation test that checks for undefined variables!